### ISSUE TITLE
System.Data.SqlClient.ManualTesting.Tests taking too long on Outerloop

### ISSUE THREAD

**Issue by pjanotti** [dotnet/corefx#30930]
Per readme it actually require some setup so likely it should be just automatically skipped, anyway, it is taking a long time on outerloop and affecting code coverage runs (see #30922).

```
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestSetPolicyWithVariations(description: "Test policy with Auto (miXedcase)", Params: ["auTo"]) finished in 321331ms.
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestSetPolicyWithVariations(description: "Test policy with Auto (Pascalcase)", Params: ["Auto"]) finished in 350376ms.
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestSetPolicyWithVariations(description: "Test policy with Always (lowercase)", Params: ["alwaysblock"]) finished in 379412ms.
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestSetPolicyWithVariations(description: "Test policy with Always (miXedcase)", Params: ["aLwAysBlock"]) finished in 408452ms.
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestSetPolicyWithVariations(description: "Test policy with Always (Pascalcase)", Params: ["AlwaysBlock"]) finished in 437494ms.
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestSetPolicyWithVariations(description: "Test policy with Never (lowercase)", Params: ["neverblock"]) finished in 466537ms.
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestSetPolicyWithVariations(description: "Test policy with Never (miXedcase)", Params: ["neVeRblock"]) finished in 495578ms.
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestSetPolicyWithVariations(description: "Test policy with Never (Pascalcase)", Params: ["NeverBlock"]) finished in 524619ms.
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestNonAzureBlockingPeriod(description: "NonAzure with Default Policy must Enable blocking", Params: ["nonexistanceserver"]) finished in 553664ms.
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestNonAzureBlockingPeriod(description: "NonAzure with Auto Policy must Enable Blocking", Params: ["nonexistanceserver", Auto]) finished in 569252ms.
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestNonAzureBlockingPeriod(description: "NonAzure with Always Policy must Enable Blocking", Params: ["nonexistanceserver", AlwaysBlock]) finished in 599642ms.
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestNonAzureBlockingPeriod(description: "NonAzure with Never Policy must Disable Blocking", Params: ["nonexistanceserver", NeverBlock]) finished in 630055ms.
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestNonAzureBlockingPeriod(description: "NonAzure (which contains azure endpoint - nonexist"..., Params: ["nonexistance.windows.net"]) finished in 660447ms.
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.PoolBlockPeriodTest.TestNonAzureBlockingPeriod(description: "NonAzure (which contains azure endpoint - nonexist"..., Params: ["nonexistance.database.windows.net.else"]) finished in 689498ms.
     System.Data.SqlClient.ManualTesting.Tests.SqlFileStreamTest.ReadFilestream [SKIP]
        Condition(s) not met: \"IsFileStreamEnvironmentSet\", \"IsIntegratedSecurityEnvironmentSet\", \"AreConnectionStringsSetup\"
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.SqlFileStreamTest.ReadFilestream finished in 718565ms.
     System.Data.SqlClient.ManualTesting.Tests.SqlFileStreamTest.OverwriteFilestream [SKIP]
        Condition(s) not met: \"IsFileStreamEnvironmentSet\", \"IsIntegratedSecurityEnvironmentSet\", \"AreConnectionStringsSetup\"
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.SqlFileStreamTest.OverwriteFilestream finished in 718565ms.
     System.Data.SqlClient.ManualTesting.Tests.SqlFileStreamTest.AppendFilestream [SKIP]
        Condition(s) not met: \"IsFileStreamEnvironmentSet\", \"IsIntegratedSecurityEnvironmentSet\", \"AreConnectionStringsSetup\"
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.SqlFileStreamTest.AppendFilestream finished in 718566ms.
```

**Comment by stephentoub** [comment]
Most of these look to be addressed with https://github.com/dotnet/corefx/pull/30856.

**Comment by stephentoub** [comment]
This output is strange:
```
     System.Data.SqlClient.ManualTesting.Tests.SqlFileStreamTest.ReadFilestream [SKIP]
        Condition(s) not met: \"IsFileStreamEnvironmentSet\", \"IsIntegratedSecurityEnvironmentSet\", \"AreConnectionStringsSetup\"
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.SqlFileStreamTest.ReadFilestream finished in 718565ms.
     System.Data.SqlClient.ManualTesting.Tests.SqlFileStreamTest.OverwriteFilestream [SKIP]
        Condition(s) not met: \"IsFileStreamEnvironmentSet\", \"IsIntegratedSecurityEnvironmentSet\", \"AreConnectionStringsSetup\"
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.SqlFileStreamTest.OverwriteFilestream finished in 718565ms.
     System.Data.SqlClient.ManualTesting.Tests.SqlFileStreamTest.AppendFilestream [SKIP]
        Condition(s) not met: \"IsFileStreamEnvironmentSet\", \"IsIntegratedSecurityEnvironmentSet\", \"AreConnectionStringsSetup\"
  WARNING: Long running test System.Data.SqlClient.ManualTesting.Tests.SqlFileStreamTest.AppendFilestream finished in 718566ms.
```
It's saying the tests are being skipped but then saying they're long-running?

**Comment by pjanotti** [comment]
I didn't notice that... it is somehow a side effect of the `PoolBlockPeriodTest` tests: if I disable the 3 theories from this class then these 3 are just skipped very quickly. Anyway closing this one since it was fixed by #30856
